### DISCUSSION TITLE
How to response by different dns server from different client ip?

### DISCUSSION THREAD

**Discussion by fcying** [coredns/coredns#4940] [Q&A] [answered]
e.g.
client ip in 10.10.1.0/24 respond from 1.1.1.1
client ip in 10.10.2.0/24 respond from 8.8.8.8

Is there any plug can support this?

**Comment by chrisohaver** [comment, marked as answer]
No built-in plugins can do it. But for an external plugin, there is https://github.com/infobloxopen/fwdpolicy in combination with https://github.com/chrisohaver/conditional

**Reply by fcying** [reply]
This solved my problem, thanks

**Comment by chrisohaver** [comment]
FWIW, CoreDNS has since (in v1.10.0) added the [view](https://github.com/coredns/coredns/tree/master/plugin/view) plugin, which enables split-dns solution such as this natively without external plugins.
